bpf/aya-bpf/src/helpers.rs

```diff
@@ -322,6 +326,114 @@ pub unsafe fn bpf_probe_read_user_str(src: *const u8, dest: &mut [u8]) -> Result
     Ok(len as usize)
 }
 
+/// Returns a null-terminated byte slice read from _user space_ stored at `src`.
+///
+/// Providing a `dest` buffer is required and using a PerCpuArray for it is
+/// recommended.
+///
+/// In case the length of `dest` is smaller then the length of `src`, an error
+/// will be returned.
+///
+/// # Arguments
```

Comment:
We use the rust std doc convention, which doesn't use a separate Arguments section. Arguments should be described in prose as part of the doc, eg:

```
 Returns a byte slice read from the _user space_ address `src`.

Reads at most `dest.len()` bytes from the `src` address, truncating if the length of the source string is larger than `dest`. On success, the destination buffer is always null terminated, and the returned slice includes the bytes up to and not including NULL.
```